src/functionalTest/resources/application-functional.yaml

```diff
@@ -21,6 +21,7 @@ exui.role.pui-user-manager: pui-user-manager
 exui.role.pui-organisation-manager: pui-organisation-manager
 exui.role.pui-finance-manager: pui-finance-manager
 exui.role.pui-case-manager: pui-case-manager
+exui.role.pui-caa: pui-caa
 
 # resent user invite interval in minutes
```

Comment:
not urgent... but can we change this to yaml format?

Reply:
in the next PR i will do